packages/api-client-core/src/support.ts

```diff
@@ -203,12 +203,18 @@ export const get = (object: Record<string, any> | null | undefined, path: string
 
 export const isCloseEvent = (event: any): event is CloseEvent => event?.type == "close";
 
-export const capitalize = (str: string | undefined | null) => {
-  const result = str === null || str === undefined ? "" : String(str);
-  return result.charAt(0).toUpperCase() + result.slice(1);
+export const capitalize = (str: string | undefined | null): string => {
+  if (typeof str !== "string") return "";
+  return camelize(str, true);
 };
 
 export const camelize = (term: string, uppercaseFirstLetter = true) => {
+
+  const capitalize = (str: string) => {
```

Comment:
Its gonna be slightly faster to not redefine this function every time the outer function executes, can you move it out to the module level?